### PR TITLE
feat(ST-75): add classroom banner

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
## Description of Change

- Added a banner component
- Added edit & delete dropdown menu
- Added classroom details in a dropdown for mobile and tablet

## Associated Ticket Link(s)

- https://trello.com/c/Ji2zj1K1

## Related Pull Request(s)

- N/A

## Screenshots / Screen Recordings
Web View:
Screen Shot 2024-09-12 at 3 22 15 PM

Mobile View:
Screen Shot 2024-09-12 at 3 21 55 PM

### CODE REVIEW

File: modules/Classroom/containers/ClassroomDetailsContainer/ClassroomDetailsContainer.tsx [hunk 1]
```diff
@@ -79,6 +80,10 @@ const ClassroomDetailsContainer = () => {
           </Tabs.Tab>
         </Tabs.List>
 
+        <Tabs.Panel value="stream">
+          {classroom ? <ClassroomStreamContainer classroom={classroom} /> : null}
```
Comment (ishmam-iqbal-sazim) [inline modules/Classroom/containers/ClassroomDetailsContainer/ClassroomDetailsContainer.tsx:84]: Does it make sense to show a loading placeholder when fetch isLoading?

File: modules/Classroom/containers/ClassroomStreamContainer/ClassroomStreamContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
```
Comment (ishmam-iqbal-sazim) [inline modules/Classroom/containers/ClassroomStreamContainer/ClassroomStreamContainer.tsx:1]: Can remove this import

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comments

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved pending passing lint check

**Pull request merged**